src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.jdbc;
 
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
+        Injector injector = Injector.getInstance("mate.jdbc");
+        ManufacturerDao manufacturerDao = (ManufacturerDao)
+                injector.getInstance(ManufacturerDao.class);
+
+        manufacturerDao.create(new Manufacturer("Lamborghini", "Italy"));
+        manufacturerDao.create(new Manufacturer("Ferrarii", "Italy"));
+        manufacturerDao.create(new Manufacturer("Porsche", "Germany"));
+
+        System.out.println("At first all manufacturers are:");
+        manufacturerDao.getAll().forEach(System.out::println);
+
+        Optional<Manufacturer> optionalManufacturer = manufacturerDao.get(34L);
+        optionalManufacturer.get().setName("Ferrari");
+        manufacturerDao.update(optionalManufacturer.get());
+
+        System.out.println("After Ferrari updating manufacturers are:");
+        manufacturerDao.getAll().forEach(System.out::println);
+
+        manufacturerDao.delete(2L);
```

Comment:
```suggestion
        System.out.println(manufacturerDao.delete(2L));
```
It's better to see the result of the operation. Also, remember how auto-increment works in DB.